tests/models/plants/test_plants_model.py

```diff
@@ -21,11 +23,17 @@ def test_PlantsModel__init__(plants_data, flora):
     assert len(plants_model.flora) == len(flora)
     assert len(plants_model.communities) == plants_data.grid.n_cells
 
-    # Check the canopy has been initialised and updated
-    assert "layer_heights" in plants_data
-    assert "leaf_area_index" in plants_data
-    assert plants_data["layer_heights"].sum() == (30 + 20 + 10) * 4
-    assert plants_data["leaf_area_index"].sum() == 3 * 4
+    # Check the canopy has been initialised and updated with some simple test sums
+    expected_layers = [
```

Comment:
As above, there are a few more places where I'd use tuples rather than lists. It's a minor nit, but I'd avoid creating lists unless you have to (i.e. you need something that's mutable).